qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -24,10 +24,10 @@
 import numpy as np
 from scipy import linalg
 
-from qiskit.algorithms.eigensolvers import EigensolverResult
-from qiskit.algorithms.list_or_dict import ListOrDict as ListOrDictType
-from qiskit.algorithms.minimum_eigensolvers import MinimumEigensolver
-from qiskit.algorithms.observables_evaluator import estimate_observables
+from qiskit_algorithms.eigensolvers import EigensolverResult
+from qiskit_algorithms.list_or_dict import ListOrDict as ListOrDictType
+from qiskit_algorithms.minimum_eigensolvers import MinimumEigensolver
+from qiskit_algorithms.observables_evaluator import estimate_observables
```

Comment:
I will just comment here since this is a bit different - the eigensolvers and minimum_eigensolvers, yes drop them, The others, well I can see estimate_observables in importable direct from qiskit_algorithms  rather than from the module, but I do not see that for ListOrDict and actually neither are in the algorithm docs from what I can see.

Reply:
I think I will leave the `estimate_observables` for now, too. The others I will drop in accordance with your other comments, too